src/components/HistoryTagsAndDifficultyTextChange/HistoryTagsAndDifficultyTextChange.tsx

```diff
@@ -0,0 +1,32 @@
+import { nullable, Text } from '@taskany/bricks';
+import styled from 'styled-components';
+
+import { tr } from './HistoryTagsAndDifficultyTextChange.i18n';
+
+interface HistoryChangeProps<T> {
+    from?: T | null;
```

Comment:
Can it be `string | null`? What other types can be used here?

Reply:
Of course you're right